pytest.ini

```diff
@@ -66,3 +66,5 @@ markers =
         federation: Tests for rabbitmq federation communication
         shovel: Tests for rabbitmq shovel communication
         contrib: tests for community-contributed agents
+
+asyncio_mode = auto
```

Comment:
Because this is a new switch can you document what this does for us?

Reply:
This allows us to run tests on asyncio code with pytest. I will add a comment.